pyvrp/cpp/Individual_bindings.cpp

```diff
@@ -44,14 +54,13 @@ PYBIND11_MODULE(_Individual, m)
             },
             py::arg("memo"))
         .def("__hash__",
-             [](Individual const &individual)
-             { return std::hash<Individual>()(individual); })
+             [](Individual const &individual) {
+                 return std::hash<Individual>()(individual);
+             })
```

Comment:
I’ve had some issues like this with the clang formatting. It seems that pre-commit doesn’t catch formatting differences between newer versions clang-format and the one that’s in the pre-commit configuration. 

Should we bump the clang-format version to the newest one?

Reply:
I'll open an issue for this. We definitely need to figure out why the pre-commit's clang-format seems to be letting stuff like this go through.

Reply:
See #166.